cmd/download.go

```diff
@@ -132,7 +132,7 @@ func runDownload(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 		}
 	}
 
-	solution := workspace.Solution{
+	metadata := workspace.Metadata{
```

Comment:
Something about this type is off to me , but I can't put my finger on why.  It may have to do with the two properties `IsRequester` and `Team.